spec/model/callbacks_spec.rb

```diff
@@ -21,10 +21,12 @@
 
     context "when using a symbol callback" do
       before do
-        class Foo::User
-          before_save :alter_name
-          def alter_name
-            name.upcase!
+        module Foo
```

Comment:
agree with keeping this rule and replacing the class definitions with [spawn_model](https://github.com/remiprev/her/blob/e2605655915129c7a7664b6518d64aee25c4827e/spec/model/associations/association_proxy_spec.rb#L15) instead.